### ISSUE TITLE
Docs for 3.2.1 and 3.3.0 are missing

### ISSUE THREAD

**Issue by player-03** [HaxeFoundation/api.haxe.org#16]
Both produce a 404:

https://api.haxe.org/v/3.2.1/
https://api.haxe.org/v/3.3/

**Comment by kLabz** [comment]
Adding `4.1.4` and `4.1.5` to that list.

**Comment by cedx** [comment]
And `4.3.0` 😄

**Comment by kLabz** [comment]
Yeah that one we know about; going back to work tomorrow, will handle that sometime this week.

**Comment by arnabp** [comment]
Looks like `3.4.7` is also missing

**Comment by kLabz** [comment]
Uh. There was a disk space issue, not sure who handled that but seems like they removed 3.4.7 :/

**Comment by arnabp** [comment]
So is it getting restored or is it lost forever?
